.ci/scripts/test_yolo12.sh

```diff
@@ -0,0 +1,198 @@
+#!/bin/bash
```

Comment:
Does this need to be running in CI?

Reply:
No my question is this script is not actually running in ci. If so why is it inside .ci folder